<ui version="4.0" >
 <class>QtInformationstagClass</class>
 <widget class="QWidget" name="QtInformationstagClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>626</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>952</width>
    <height>626</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>952</width>
    <height>626</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Informationstag</string>
  </property>
  <widget class="QLabel" name="labelInformationstag" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>931</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text" >
    <string>Informationstag</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btBeenden" >
   <property name="geometry" >
    <rect>
     <x>830</x>
     <y>590</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Beenden</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBoxAngebot" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>80</y>
     <width>481</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title" >
    <string>Angebote</string>
   </property>
   <widget class="QLabel" name="labelNameAngebot" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMin" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>120</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Minimal:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>90</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Maximal:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameAngebot" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBoxMin" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>120</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="value" >
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBoxMax" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>90</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>2</number>
    </property>
    <property name="value" >
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btAngebotHinzufuegen" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Hinzuf체gen</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableAngebote" >
    <property name="geometry" >
     <rect>
      <x>230</x>
      <y>20</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount" >
     <number>4</number>
    </property>
    <column>
     <property name="text" >
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Angebot</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Min</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Max</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btVorlageHinzufuegen" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>20</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Vorlage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btAngebotEntfernen" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>150</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Entfernen</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBoxTeilnehmer" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>280</y>
     <width>491</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title" >
    <string>Teilnehmer</string>
   </property>
   <widget class="QLabel" name="labelNameTeilnehmer" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameTeilnehmer" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>70</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btTeilnehmerHinzufuegen" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>290</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Hinzuf체gen</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableTeilnehmer" >
    <property name="geometry" >
     <rect>
      <x>230</x>
      <y>20</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount" >
     <number>8</number>
    </property>
    <column>
     <property name="text" >
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Teilnehmer</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Wunsch 1</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Wunsch 2</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Wunsch 3</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Wunsch 4</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Wunsch 5</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Wunsch 6</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btTeilnehmerEntfernen" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>290</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Entfernen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btZufaellig1" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Zuf채llig (1)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btZufaellig10" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>30</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Zuf채llig (10)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBoxVeranstaltungen" >
   <property name="geometry" >
    <rect>
     <x>690</x>
     <y>80</y>
     <width>251</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title" >
    <string>Veranstaltungen</string>
   </property>
   <widget class="QTreeWidget" name="treeVeranstaltungen" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
    <column>
     <property name="text" >
      <string>1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btVeranstaltungenEntfernen" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>460</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Entfernen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btPlanErstellen" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Plan erstellen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBoxOptimal" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>30</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Optimal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btAbbrechen" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>60</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Abbrechen</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette" >
     <palette>
      <active>
       <colorrole role="WindowText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>139</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>139</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>119</red>
          <green>119</green>
          <blue>121</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Berechnung l채uft...</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBoxEinstellungen" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title" >
    <string>Einstellungen</string>
   </property>
   <widget class="QPushButton" name="btEinstellungenVerwenden" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>110</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Verwenden</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBoxAnzWuensche" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>70</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>20</number>
    </property>
    <property name="value" >
     <number>6</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sBoxAnzTermine" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>10</number>
    </property>
    <property name="value" >
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelAnzahlTermine" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>52</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Termine:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAnzahlWuensche" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>W체nsche:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btAllesNeu" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>150</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Alles Neu</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBoxPrioritaeten" >
   <property name="geometry" >
    <rect>
     <x>510</x>
     <y>280</y>
     <width>171</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title" >
    <string>Priorit채ten</string>
   </property>
   <widget class="QPushButton" name="btPrioritaetenVerwenden" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>290</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Verwenden</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>931</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Ein Beitrag zum 26. Bundeswettbewerb Informatik von Leon Sering</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>btBeenden</sender>
   <signal>clicked()</signal>
   <receiver>QtInformationstagClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>638</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel" >
     <x>109</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
